VizbeeManager.js

```diff
@@ -113,6 +136,28 @@ class VizbeeManager {
     removeAllListeners(eventName) {
         VizbeeNativeEmitter.removeAllListeners(eventName)
     }
+
+    //------------------
+    // Sign in Helpers
+    //------------------
+
+    registerSignInListeners() {
+        // listen for sign in events
+        this.addListener(
+            "VZB_INVOKE_GET_SIGNIN_INFO",
+            this.invokeAndHandleGetSignInInfo,
+            this
+        );
+    }
+    
+    invokeAndHandleGetSignInInfo() {
+        if (this.signInDelegate) {
+            let signInInfo = this.signInDelegate.getSignInInfo();
+            if (signInInfo instanceof VizbeeSignInInfo) {
```

Comment:
Should we do any other checks on the signInInfo? Are any of these properties to be defined?

Reply:
For the initial release the above should be sufficient since each customers implement specific properties we are not sure which ones to make mandatory and validate them